docs/glsl-fuzz-walkthrough.md

```diff
@@ -603,6 +612,48 @@ the reduction of this variant leads to the following files:
   reduction. It typically refers to the smallest shader the reducer could obtain
   for this particular reduction.
 
+## Running shaders from the command line
+
+Sometimes it can be useful to run individual shaders
+from the command line.
+
+This can be achieved using the `run-shader-family` script,
+which runs a single shader or a shader family
+on a device, via the server.
+For example:
+
+```sh
+# Run one shader
+run-shader-family variant_001.json --server http://localhost:8080 --worker pixel3
+
+# Run shader family directory `bubblesort_family`
+run-shader-family bubblesort_family --output results/ --server http://localhost:8080 --worker pixel3
+```
+
+The results will be output to the current directory by default, or to the directory specified
+by `--output`.
+The output `.txt` files contain the run log for each shader.
+
+### vkrun
+
+As described earlier,
+the Vulkan worker uses `vkrun`
+to run SPIR-V shaders on the device.
+You can use this script directly
+if you want to re-run some SPIR-V shaders.
+Your device must show in `adb devices`.
+For example:
+
+```sh
+vkrun test.vert variant_005.frag variant_005.json --serial ABCD
+```
+
```

Comment:
It's a little confusing above: we say that vkrun enables running SPIR-V shaders on the device, but then the example of how to use vkrun involves a .frag file.  We should say that you can use this to run a single .frag shader on a Vulkan device, via translation to .spv, and make it clear that the convenience here over run-shader-family is that it does not require a server.

Does vkrun have an option where you can give it a .spv file instead of a .frag file?  If so it would be good to document that too.